ADRs/RH/SIG-SRE/ADR-00007-Error-Budget-Policy.md

```diff
@@ -0,0 +1,250 @@
+  </tr>
+  <tr>
+   <td>Not Met</td>
+   <td>Low</td>
+   <td>High</td>
+   <td>Review the SLO. It is likely a false positive and needs to be adjusted (or even deleted).</td>
+   <td>Focus on "feature work". However, fixing the SLO is a priority, as it may hide actual issues.</td>
+  </tr>
+  <tr>
+   <td>Not Met</td>
+   <td>High</td>
+   <td>High</td>
+   <td>Review the SLO. It could be a false positive and needs to be adjusted (or even deleted). If this is causing toil for another team then fix it.</td>
+   <td>Balance the velocity of "reliability work" vs. "feature work" based on all inputs, <em>considering the effect on the affected teams and the risk of error.</em></td>
+  </tr>
+  <tr>
+   <td>Not Met</td>
+   <td>Low</td>
+   <td>Low</td>
+   <td>If there is no SLO reflecting customer dissatisfaction, create it if appropriate.<p>If an SLO exists, then review based on customer feedback.<p>Fix the Service if needed.</td>
+   <td>Focus on "reliability work" to resolve the customer satisfaction problem.</td>
+  </tr>
+  <tr>
+   <td>Not Met</td>
+   <td>High</td>
+   <td>Low</td>
+   <td>Offload toil and fix product and/or improve automated fault mitigation.</td>
+   <td>Stop "feature work" and focus on "reliability work" to resolve the customer satisfaction problem.</td>
+  </tr>
+</table>
+
+# Review Cadence
+
+It is absolutely necessary to setup a cadence for a regular review of the state of the error budget consumption at any point in time. The following is an example of what would be considered the minimum viable cadence required to keep on top of "burning" error budget. These meetings are an essential part of the "Iteration" phase as described in the [SLO Lifecycle ADR](https://github.com/operate-first/sre/pull/13). The reviews refer to these [personas](https://github.com/operate-first/sre/blob/main/ADRs/RH/SIG-SRE/ADR-00002%20Personas%20related%20to%20Managed%20Services.md).
+
+## Weekly Team Uptime Review
+
+The team/s (SRE/Engineering) responsible for the services need to collaboratively review the availability of the services on a regular basis. As per many typical engineering processes that have been well establish, this can, and should, include regular sprint reviews (standups) and retrospectives. However, it has proven very beneficial to have a regular (e.g. weekly) review between, at least, SRE and engineering (but also possibly including the Product Owner). SRE, for example, are typically those who first identify that the error budget has been consumed, or is being consumed. At this meeting, they have the opportunity to inform the engineering team. If the error budget consumption coincides with an incident, this is often in the form of an RCA (Root Cause Analysis) document. The engineering team receive this information, and in close collaboration with the Product Owner, they can prioritise the rectification of the issue/s.
```

Comment:
```suggestion
The team/s (SRE/Engineering) responsible for the services need to collaboratively review the availability of the services on a regular basis. As per many typical engineering processes that have been well established, this can, and should, include regular sprint reviews (standups) and retrospectives. However, it has proven very beneficial to have a regular (e.g. weekly) review between, at least, SRE and engineering (but also possibly including the Product Owner). SRE, for example, are typically those who first identify that the error budget has been consumed, or is being consumed. At this meeting, they have the opportunity to inform the engineering team. If the error budget consumption coincides with an incident, this is often in the form of an RCA (Root Cause Analysis) document. The engineering team receive this information, and in close collaboration with the Product Owner, they can prioritise the rectification of the issue/s.
```

Reply:
Fixed